fe/pom.xml

```diff
@@ -839,6 +839,13 @@ under the License.
                 <version>12.4.2.jre11</version>
             </dependency>
 
+            <!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
+            <dependency>
+                <groupId>com.mysql</groupId>
+                <artifactId>mysql-connector-j</artifactId>
+                <version>9.2.0</version>
+            </dependency>
+
```

Comment:
This is GPL licensed, and we can not include it then distributed in binary package.